src/test/java/org/broadinstitute/consent/http/service/DataAccessRequestServiceTest.java

```diff
@@ -375,10 +368,52 @@ void validateProgressReport() {
     parentDar.setSubmissionDate(Timestamp.from(Instant.now()));
     parentDar.setDatasetIds(List.of(1, 2, 3));
     parentDar.setUserId(user.getUserId());
+    assertDoesNotThrow(() -> service.validateProgressReport(user, progressReport, parentDar));
+  }
+
+  @Test
+  void validateProgressReportWithCollaboratorsAndStaffInSameInstitution() {
+    User user = createUserWithPrerequisites();
+    DataAccessRequest progressReport = generateProgressReport();
+    progressReport.setDatasetIds(List.of(1, 2));
+    DataAccessRequestData progressReportData = progressReport.getData();
+    Collaborator collaborator1 = new Collaborator();
+    collaborator1.setEmail("1" + user.getEmail());
+    progressReportData.setInternalCollaborators(Collections.singletonList(collaborator1));
+    Collaborator collaborator2 = new Collaborator();
+    collaborator2.setEmail("2" + user.getEmail());
+    progressReportData.setLabCollaborators(Collections.singletonList(collaborator2));
+    DataAccessRequest parentDar = generateDataAccessRequest();
+    parentDar.setSubmissionDate(Timestamp.from(Instant.now()));
+    parentDar.setDatasetIds(List.of(1, 2, 3));
+    parentDar.setUserId(user.getUserId());
+    when(userDAO.findUserByEmail(collaborator1.getEmail())).thenReturn(user);
     when(institutionService.findInstitutionForEmail(any())).thenReturn(user.getInstitution());
     assertDoesNotThrow(() -> service.validateProgressReport(user, progressReport, parentDar));
   }
 
+  @Test
+  void validateProgressReportWithCollaboratorsAndStaffInInvalidInstitution() {
+    User user = createUserWithPrerequisites();
+    DataAccessRequest progressReport = generateProgressReport();
+    progressReport.setDatasetIds(List.of(1, 2));
+    DataAccessRequestData progressReportData = progressReport.getData();
+    Collaborator collaborator1 = new Collaborator();
+    collaborator1.setEmail("1" + user.getEmail());
+    progressReportData.setInternalCollaborators(Collections.singletonList(collaborator1));
+    Collaborator collaborator2 = new Collaborator();
+    collaborator2.setEmail("2" + user.getEmail());
+    progressReportData.setLabCollaborators(Collections.singletonList(collaborator2));
+    DataAccessRequest parentDar = generateDataAccessRequest();
+    parentDar.setSubmissionDate(Timestamp.from(Instant.now()));
+    parentDar.setDatasetIds(List.of(1, 2, 3));
+    parentDar.setUserId(user.getUserId());
+    when(userDAO.findUserByEmail(collaborator1.getEmail())).thenReturn(user);
+    when(institutionService.findInstitutionForEmail(any())).thenReturn(null);
```

Comment:
I found this test challenging to trace since the collaborators are all set up with the same email domains and, in theory, **should** all match the same institution as the user. The key to the test passing is that there is a deeply nested check on the institution so this is why the test succeeds. It would be clearer to me if we had real collaborators with real emails in different orgs than the user created on line 397.